docker/browsersync/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.4
+
+MAINTAINER Jules Aaelio
+
+RUN apk add --update nodejs
+
+CMD npm install && npm start
```

Comment:
I don't see a copy of package.json in the Dockerfile, meaning this Dockerfile doesn't work on its own: the `CMD` would fail as is.
We usually want a Dockerfile to be self-contained, docker-compose is just a convenience wrapper above.